modules/standard/Subprocess.chpl

```diff
@@ -983,76 +983,99 @@ module Subprocess {
   extern const SIGQUIT: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGSEGV' is deprecated. Use 'Sys.SIGSEGV' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGSEGV' is deprecated. Use 'OS.POSIX.SIGSEGV' instead."
   extern const SIGSEGV: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGSTOP' is deprecated. Use 'Sys.SIGSTOP' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGSTOP' is deprecated. Use 'OS.POSIX.SIGSTOP' instead."
   extern const SIGSTOP: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGTERM' is deprecated. Use 'Sys.SIGTERM' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGTERM' is deprecated. Use 'OS.POSIX.SIGTERM' instead."
   extern const SIGTERM: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGTRAP' is deprecated. Use 'Sys.SIGTRAP' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGTRAP' is deprecated. Use 'OS.POSIX.SIGTRAP' instead."
   extern const SIGTRAP: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGTSTP' is deprecated. Use 'Sys.SIGTSTP' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGTSTP' is deprecated. Use 'OS.POSIX.SIGTSTP' instead."
   extern const SIGTSTP: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGTTIN' is deprecated. Use 'Sys.SIGTTIN' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGTTIN' is deprecated. Use 'OS.POSIX.SIGTTIN' instead."
   extern const SIGTTIN: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGTTOU' is deprecated. Use 'Sys.SIGTTOU' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGTTOU' is deprecated. Use 'OS.POSIX.SIGTTOU' instead."
   extern const SIGTTOU: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGURG' is deprecated. Use 'Sys.SIGURG' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGURG' is deprecated. Use 'OS.POSIX.SIGURG' instead."
   extern const SIGURG: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGUSR1' is deprecated. Use 'Sys.SIGUSR1' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGUSR1' is deprecated. Use 'OS.POSIX.SIGUSR1' instead."
   extern const SIGUSR1: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGUSR2' is deprecated. Use 'Sys.SIGUSR2' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGUSR2' is deprecated. Use 'OS.POSIX.SIGUSR2' instead."
   extern const SIGUSR2: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGXCPU' is deprecated. Use 'Sys.SIGXCPU' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGXCPU' is deprecated. Use 'OS.POSIX.SIGXCPU' instead."
   extern const SIGXCPU: c_int;
   pragma "no doc"
-  deprecated "'Subprocess.SIGXFSZ' is deprecated. Use 'Sys.SIGXFSZ' instead."
+  pragma "last resort"
+  deprecated "'Subprocess.SIGXFSZ' is deprecated. Use 'OS.POSIX.SIGXFSZ' instead."
   extern const SIGXFSZ: c_int;
 
   // These signals are not strictly required by POSIX.1.2008 2013 edition
```

Comment:
I think this comment and the comment under it can just be deleted now.  It also looks like the documentation for `subprocess.sendPosixSignal` should be updated.